distributed/tests/test_profile.py

```diff
@@ -46,6 +47,45 @@ def test_f():
     assert 95 < g['count'] + h['count'] <= 100
 
 
+@pytest.mark.skipif(
+        sys.version_info < (3, 5),
+        reason='stacktrace not supported on python < 3.5')
```

Comment:
Perhaps instead we should have the following line within the function

```python
pytest.importorskip('stacktrace`)
```

That way if stacktrace does support lower versions in the future, or if we don't have stacktrace installed, then things will still work correctly.

Reply:
That's a much better idea, I'll do that. Thanks for the suggestion.